config/cluster-baremetal-operator/cluster-baremetal-operator.yaml

```diff
@@ -20,6 +20,29 @@ spec:
       priorityClassName: system-node-critical
       serviceAccountName: cluster-baremetal-operator
       containers:
```

Comment:
so personally i'd prefer these changes in the config/promethus directory

Reply:
Done.